src/main.rs

```diff
@@ -48,9 +48,11 @@ enum Mode {
     path: PathBuf,
   },
   /// Compiles the program and runs it with the Rust HVM implementation.
-  Run(RunArgs),
+  RunRs(RunArgs),
   /// Compiles the program and runs it with the C HVM implementation.
   RunC(RunArgs),
+  /// Compiles the program and runs it with the C HVM implementation.
+  Run(RunArgs),
```

Comment:
Since run and run-c are doing the same thing you can remove it and specify run as an alias for run-c

Reply:
[6380803](https://github.com/HigherOrderCO/Bend/pull/630/commits/6380803dae279d388300b3bc0524d86a11c604ae)
Thanks for the feedback! Added the change to this commit.